src/plugins/about/index.js

```diff
@@ -0,0 +1,133 @@
+const log = require('Debug')('log:about');
+const trace = require('Debug')('trace:about');
+const debug = require('Debug')('debug:about');
+
+const bluebird = require('bluebird');
+const Listener = require('Listener');
+const execFile = require('child_process').execFile;
+//bluebird.promisify(execAsync);
+
+
+//private functions
+
+
+function execAsync(command,args){
```

Comment:
Wondering why a custom implementation of execAsync was made here instead of using the promisify'd version? Was there a problem with Bluebird's wrapper?